src/features/movie/components/overview/Overview.test.tsx

```diff
@@ -0,0 +1,35 @@
+import { render, screen } from "@testing-library/react";
+import { MemoryRouter } from "react-router-dom";
+import { Overview } from "./Overview";
+
+const movie = {
```

Comment:
You're repeating mock data. Can you do it better? ;)

Reply:
Same as above